Decred Wallet/Features/Send/SendViewController.swift

```diff
@@ -369,30 +369,30 @@ extension SendViewController {
 
     func calculateAndDisplayUSDAmount() {
         guard let dcrAmount = Double(self.amountTextField.text ?? ""), let exchangeRate = self.exchangeRate else {
-            self.usdAmountLabel.text = "- USD"
+            self.usdAmountLabel.text = "0 USD"
             return
         }
 
         let usdAmount = NSDecimalNumber(value: dcrAmount).multiplying(by: exchangeRate)
-        self.usdAmountLabel.text = "\(usdAmount.round(2)) USD"
+        self.usdAmountLabel.text = "\(usdAmount.round(2).formattedWithSeparator) USD"
```

Comment:
#645 says usd amount should always be prefixed with `$`. Can do away with the `USD` suffix.